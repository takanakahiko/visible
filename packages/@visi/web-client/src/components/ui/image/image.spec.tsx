import { render } from '@testing-library/react';
import React from 'react';

import { Image } from '.';

describe('Image', () => {
  it('matches snapshot', () => {
    const { container } = render(<Image src="/image.png" alt="image" />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
