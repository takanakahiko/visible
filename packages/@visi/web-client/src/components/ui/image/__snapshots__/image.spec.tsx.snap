// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image matches snapshot 1`] = `
<img
  alt="image"
  class="object-contain"
  src="/image.png"
/>
`;
